src/TaxToken.sol

```diff
@@ -494,4 +494,18 @@ contract TaxToken {
         }
     }
 
+    /// @notice This function is a VIEW function that returns the amount of industry tokens,
+    ///         full balance, and normal tokens a wallet has.
+    /// @dev    This function is for the front end to pull industry data for a specific wallet.
+    /// @return numTokens the amount of taxTokens the _wallet holds.
+    /// @return numIndustryTokens the amount of tokens they hold that is industry tokens.
+    /// @return numDifference the amount of tokens they have that are NOT industry tokens.
+    function getIndustryTokens(address _wallet) external view returns (uint numTokens, uint numIndustryTokens, uint numDifference) {

```

Comment:
Lets add the lifetime amount as well at the end of this so we can just use the one getter for any use.